handshake/crypto_setup_server.go

```diff
@@ -193,7 +194,6 @@ func (h *cryptoSetupServer) Open(dst, src []byte, packetNumber protocol.PacketNu
 		if err == nil {
 			if !h.receivedForwardSecurePacket { // this is the first forward secure packet we receive from the client
 				h.receivedForwardSecurePacket = true
-				close(h.aeadChanged)
```

Comment:
Just to be clear, this change removes the additional RTT (waiting for the client to send its first FW packet) you describe in the PR msg? If we didn't change this, we would be correct, but with an additional RTT?

Reply:
No. We already returned the session as soon as the encryption level was forward-secure in https://github.com/lucas-clemente/quic-go/blob/master/server.go#L307. We never waited for the handshake to complete (which, according to the old definition, was when we received the first forward-secure packet from the client).

Reply:
Ok, I thought we were waiting for the chan to be closed.

Reply:
But you are right that this changes our definition of handshake completeness. This also affects the timeout. Do we want that?

Reply:
I don't have a strong opinion about the timeout.

Once we move to TLS 1.3, we will have to wait for the TLS Finished message (which would be roughly equivalent to receiving the forward-secure packet now). The question is, when do we want to return a connection in `Listener.Accept`?